doc/tutorials/app/orbbec_astra.markdown

```diff
@@ -43,33 +43,6 @@ In order to use the Astra camera's depth sensor with OpenCV you should do the fo
     @endcode
     If the above two variables are empty, then you need to source `OpenNIDevEnvironment` again.
 
-    @note Orbbec OpenNI SDK version 2.3.0.86 and newer does not provide `install.sh` any more.
-    You can use the following script to initialize environment:
-    @code{.text}
-    # Check if user is root/running with sudo
-    if [ `whoami` != root ]; then
-        echo Please run this script with sudo
-        exit
-    fi
-
-    ORIG_PATH=`pwd`
-    cd `dirname $0`
-    SCRIPT_PATH=`pwd`
-    cd $ORIG_PATH
-
-    if [ "`uname -s`" != "Darwin" ]; then
-        # Install UDEV rules for USB device
-        cp ${SCRIPT_PATH}/orbbec-usb.rules /etc/udev/rules.d/558-orbbec-usb.rules
-        echo "usb rules file install at /etc/udev/rules.d/558-orbbec-usb.rules"
-    fi
-
-    OUT_FILE="$SCRIPT_PATH/OpenNIDevEnvironment"
-    echo "export OPENNI2_INCLUDE=$SCRIPT_PATH/../sdk/Include" > $OUT_FILE
-    echo "export OPENNI2_REDIST=$SCRIPT_PATH/../sdk/libs" >> $OUT_FILE
-    chmod a+r $OUT_FILE
-    echo "exit"
-    @endcode
```

Comment:
The note removal is not correct. I just downloaded fresh OpenNI SDK for my Orbbec Astra Pro camera (OpenNI_2.3.0.86_202210111154_4c8f5aa4_beta6_linux) and the script is not there. There is only setup script for udev rules. In case if you drop the note, we need put `OPENNI2_INCLUDE` and `OPENNI2_REDIST` environment variables to OpenCV build steps to use the SDK.

Reply:
I have tested the latest version 2.3.0.86. Astra Pro doesn't work with it on laptop or dev board (and didn't get reply from Orbbec either). That's why I put the link of the older working version of SDK which doesn't need the extra step described in the note and removed the note.
> The SDK can be accessed via Orbbec's 3D club website. We can use the link instead of GDrive.

Reply:
I propose to mention `OPENNI2_INCLUDE` and `OPENNI2_REDIST` environment variables and how OpenNI is found.